docs/yaml-config.rst

```diff
@@ -0,0 +1,98 @@
+Read the Docs YAML Config
+=========================
+
+Read the Docs now has support for configuring builds with a YAML file.
+The file, 
+`readthedocs.yml`,
+must be in the root directory of your project.
+
+.. warning:: This feature is in a beta state.
+             Please file an Issue if you find anything wrong.
```

Comment:
spurious proper capitalization, might be good to link issue tracker or docs on submitting issue if we have that.